runtime/src/accounts_db.rs

```diff
@@ -3728,6 +3730,12 @@ impl AccountsDb {
             |loaded_account: LoadedAccount,
              accum: &mut Vec<Vec<CalculateHashIntermediate>>,
              slot: Slot| {
+                let pubkey = *loaded_account.pubkey();
+                let rng_index = pubkey.as_ref()[0] as usize * bins / max_plus_1;
```

Comment:
nit: how about renaming this to `rng_bin_index`, to signal we are mapping the pubkey to a `bin` index in the range `0...bins`